src/components/Editor/MediaUploader/LocalUploader.jsx

```diff
@@ -89,7 +89,7 @@ const LocalUploader = ({
       }));
 
       setPendingUploads([]);
-      uploadedFiles.forEach(file => insertImageToEditor(file));
+      uploadedFiles.forEach(file => insertMediaToEditor(file));
```

Comment:
```suggestion
      uploadedFiles.forEach(insertMediaToEditor);
```
Shouldn't this work?